beacon-light-client/plonky2/input_fetchers/balance_verification/withdrawal_credentials/runnable/diva_scheduler.ts

```diff
@@ -36,15 +41,18 @@ const snapshot = new ethers.Contract(
   provider,
 );
 
-const validatorManager = new ethers.Contract(
-  options['validator-manager-contract-address'],
-  validatorManagerAbi,
-  provider,
-);
+const validatorManager = options['validator-manager-contract-address']
+  ? new ethers.Contract(
+      options['validator-manager-contract-address'],
+      validatorManagerAbi,
+      provider,
+    )
+  : undefined;
 
 snapshot.on('SnapshotTaken', async (_: number, currentSlot: number) => {
-  const withdrawalCredentials =
-    await validatorManager.getWithdrawalCredentials();
+  const withdrawalCredentials = validatorManager
+    ? await validatorManager.getWithdrawalCredentials()
+    : options['withdrawal-credentials'];
```

Comment:
This is hacky. Write an IIFE that returns the withdrawal credentials and branch on options['validator-manager-contract-address'] !== undefined.